examples/04-reverse-proxy/main.go

```diff
@@ -15,12 +14,24 @@ import (
 	"github.com/borderzero/border0-go/listen"
 )
 
+// This is the code we inject.
+// Just for demonstration purposes, we're replacing the title of the page with a blinking text.
 const appendScript = `
+<style>
+  .blink {
+    animation: blinker 1s linear infinite;
+  }
+  @keyframes blinker {
+    50% {
+      opacity: 0;
+    }
+  }
+</style>
 <script>
 	const replaceTitleInterval = setInterval(() => {
 		const title = document.querySelector('.module--header h2 span');
 		if (title) {
-			title.innerHTML = 'Welcome to Border0'
+			title.innerHTML = '<span class="blink">Welcome to Border0</span><br><marquee bgcolor="Green" direction="left" >Your own version of the BBC</marquee>'
```

Comment:
`direction="left" >` => `direction="left">`, there is an extra space between `"` and `>`